websockets.md

```diff
@@ -45,12 +45,12 @@ wss://{host_name}/text-to-speech/api/v1/synthesize
 ```
 {: codeblock}
 
-where `{host_name}` identifies the regional host for your application:
+where `{host_name}` is the location in which your application is hosted:
 
--   `stream.watsonplatform.net` for US South and UK (the following examples use this host name)
--   `stream-fra.watsonplatform.net` for Germany
--   `gateway-syd.watsonplatform.net` for Sydney and AP North
--   `gateway-wdc.watsonplatform.net` for US East
+-   `stream.watsonplatform.net` for Dallas (the following examples use this host name)
+-   `stream-fra.watsonplatform.net` for Frankfurt
+-   `gateway-syd.watsonplatform.net` for Sydney
+-   `gateway-wdc.watsonplatform.net` for Washington, DC
```

Comment:
The API ref for IBM Cloud still shows London in both the service endpoint and wss sections. London is still valid for older instances (and still appears in the GUI even now).